src/Features/CSharp/Portable/EditAndContinue/BreakpointSpans.cs

```diff
@@ -58,26 +58,52 @@ private static bool IsBlank(TextLine line)
         /// Given a syntax token determines a text span delimited by the closest applicable sequence points 
         /// encompassing the token.
         /// </summary>
+        /// <param name="minLength">
+        /// In case there are multiple breakpoint spans starting at the given <paramref name="position"/>,
+        /// <paramref name="minLength"/> can be used to disambiguate between them. 
+        /// The inner-most available span whose length is at least <paramref name="minLength"/> is returned.
```

Comment:
`minLength` is only used to disambiguate two spans that start at the same point: "In case there are multiple breakpoint spans starting at the given <paramref name="position"/> ..."